jenkins-pipelines/upgrade-with-raft/rolling-upgrade-ubuntu22.04-with-raft.jenkinsfile

```diff
@@ -0,0 +1,16 @@
+#!groovy
+
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+rollingUpgradePipeline(
+    backend: 'gce',
+    base_versions: '',  // auto mode
+    linux_distro: 'ubuntu-jammy',
+    disable_raft: false,
+    use_preinstalled_scylla: true,
+    test_name: 'upgrade_test.UpgradeTest.test_rolling_upgrade',
+    test_config: 'test-cases/upgrades/rolling-upgrade.yaml',
```

Comment:
Should apply the configuration yaml file here.

Reply:
ditto